Specifications/model/information.model.md

```diff
@@ -0,0 +1,151 @@
+# Dataspace Information Model
+
+The information model defines the core concepts, entities, and relationships that underpin a `Dataspace`.
+
+## 1 Terminology
+
+### Dataspace
+
+A `Dataspace` is a set of technical services that facilitate interoperable asset sharing between entities.
+
+### DataspaceAuthority
+
+A `DataspaceAuthority` is an entity that manages a `Dataspace`.
+
+### Participant
+
+A `Participant` is a `Dataspace` member that provides and/or consumes assets.
+
+### ParticipantAgent
+
+A `ParticipantAgent` is a technology system that performs operations on behalf of a `Participant`.
+
+### IdentityProvider
+
+An `IdentityProvider` is a trusted technology system that creates, maintains, and manages identity information for a `Participant` and `ParticipantAgents`.
+
+### CredentialIssuer
+
+A `CredentialIssuer` is a trusted technology system that issues verifiable credentials for a `Participant` and `ParticipantAgents`.
+
+### ClearingHouse (TBD)
+
+A `ClearingHouse` is a trusted technology system that records and verifies domain events.
+
+### DataspaceRegistrationService
+
+A `DataspaceRegistrationService` is a technology system that maintains the state of `Participants` in a `Dataspace`.
+
+### Asset
+
+Data or a technical service that can be shared by a `Participant`.
+
+### Policy
+
+A set of rules, duties, and obligations that define the terms of use for an `Asset`.
+
+### Offer
+
+A concrete `Policy` associated with a specific `Asset`.
+
+### Agreement
+
+A concrete `Policy` associated with a specific `Asset` that has been signed by both the provider and consumer `Participants`.
+
+### Catalog
+
+A collection of entries representing `Assets` and their `Offers` that is advertised by a provider `Participant`.
+
+### CatalogService
+
+A `PariticipantAgent` that makes a `Catalog` accessible to `Participants`.
+
+### Connector (DataService)
```

Comment:
I like the way of basing the Connector definition on the ParticipantAgent rather than saying it is a [server](https://github.com/International-Data-Spaces-Association/IDS-G/tree/main/Glossary#connector). Let's enhance it a bit further to make clear whether a connector may appear as a service, as a deployed software asset, as a physical server...

Reply:
It could be any of those, but they are implementation details. I was using "technology system" to avoid over-specifying what it is.